src/backend/assets/js/main.js

```diff
@@ -203,9 +203,40 @@ $(document).ready(function () {
     });
   };
 
+  function viewsFilter(filterType, curView, date) {
+    const filter = allFilter[filterType];
+    if(curView === 'list') {
+      $(filter.dateFilter).each(function () {
+        if(date !== 'all') {
+          if ($(this).attr('class').split(' ')[0] === date) {
+            $(this).removeClass('hide');
+          } else {
+            $(this).addClass('hide');
+          }
+        } else{
+          $(this).removeClass('hide');
+        }
```

Comment:
I think it might be possible to compress this big `if` down to just two lines, decision logic and action:
```js
const showThing = date === 'all' || $(this).hasClass(date);
$(this).toggleClass('hide', !showThing);
```

(Edit: Oh I see, this is not new code, just moved!)